projects/cdk/directives/control/control.directive.ts

```diff
@@ -9,6 +9,6 @@ export class TuiControlDirective {
     constructor(@Inject(NgControl) private readonly ngControl: NgControl) {}
 
     get control(): AbstractControl {
-        return this.ngControl.control!;
+        return this.ngControl.control as AbstractControl;
```

Comment:
What difference do such changes make? Isn't it the same thing, just more verbose?